crates/red_knot_test/src/lib.rs

```diff
@@ -32,13 +32,13 @@ pub fn run(path: &Path, title: &str) {
     for test in suite.tests() {
         if let Err(failures) = run_test(&test) {
             any_failures = true;
-            println!("\n{}\n", test.name().bold().underline());
+            println!("\n{}", test.name().bold().underline());
```

Comment:
nit: I think the blank line in between the title and the line-by-line details really helps readability here; I'd keep it, personally 😄

```suggestion
            println!("\n{}\n", test.name().bold().underline());
```